src/client/components/Modal/Modal.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import './Modal.styles.css';
+import PropTypes from 'prop-types';
+
+const onCloseModal = () => {
+  document.getElementById('modalbox').className = 'modalbox';
+};
+
+const toggleModal = () => {
+  document.querySelector('.modalbox').classList.toggle('showmodal');
+};
+
+export default function ModalResponsive({
```

Comment:
I suggest splitting this into smaller components. Large functions with a lot of indentation is ususally harder to maintain. I also suggest striving for as few input parameters as possible. 3-4 at the most.